tests/integration/base.sh

```
@@ -73,29 +83,50 @@ pnpm committee:new \
 check_last_line "$SCRIPT_DIR/output/events.txt" '{"Next":10}'
 
-heading "Mock encrypted plaintext"
-$SCRIPT_DIR/lib/fake_encrypt.sh --input "$SCRIPT_DIR/output/pubkey.bin" --output "$SCRIPT_DIR/output/output.bin" --plaintext $PLAINTEXT --params "$ENCODED_PARAMS"
+if [[ "$PROOF_AGGREGATION_ENABLED" == "true" ]]; then
+  heading "Wire MockE3Program → Enclave so publishInput triggers decryption"
+  pnpm e3-program:setMockEnclave --network localhost
 
-heading "Mock publish input e3-id"
-pnpm e3-program:publishInput --network localhost  --e3-id 0 --data 0x12345678
+  heading "Encrypt plaintext under the published committee pubkey"
+  $SCRIPT_DIR/lib/fake_encrypt.sh --input "$SCRIPT_DIR/output/pubkey.bin" --output "$SCRIPT_DIR/output/output.bin" --plaintext $PLAINTEXT --params "$ENCODED_PARAMS"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)